src/test/java/uk/gov/companieshouse/missingimagedelivery/orders/api/service/DescriptionProviderServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.companieshouse.missingimagedelivery.orders.api.service;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.core.Is.is;
+
+/**
+ * Unit tests the {@link DescriptionProviderService} class.
+ */
+class DescriptionProviderServiceTest {
+
+    private static final String COMPANY_NUMBER = "00006400";
+    private static final String EXPECTED_DESCRIPTION = "missing image delivery for company " + COMPANY_NUMBER;
+
+    @Test
+    @DisplayName("Provides expected description")
+    void getDescriptionProvidesExpectedDescription() {
```

Comment:
This test seems to fail for me.

Reply:
You will need to update api-enumerations submodule.